### PR TITLE
Adding Bigtable row mutations for setting cell value.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

**Comment by dhermes** [comment]
@tseaver PTAL

File: gcloud/bigtable/row.py [hunk 1]
```diff
@@ -92,6 +99,65 @@ def _get_mutations(self, state=None):
...
+        if isinstance(value, six.integer_types):
+            value = _PACK_I64(value)
+        value = _to_bytes(value)
+        if timestamp is None:
+            # Use -1 for current Bigtable server time.
+            timestamp_micros = -1
+        else:
+            timestamp_micros = _microseconds_from_datetime(timestamp)
+            # Truncate to millisecond granularity.
+            timestamp_micros -= (timestamp_micros % 1000)
```
Comment (tseaver) [inline gcloud/bigtable/row.py]: Seriously, the protobuf field is named `timestamp_micros` but requires millisecond precision?

**Comment by tseaver** [comment]
LGTM

File: gcloud/bigtable/row.py (hunk 1 above)
Comment (dhermes) [inline gcloud/bigtable/row.py]: Yeah the granularity is [defined at `Table` level][1] but milliseconds is the only choice. I'm not quite sure what the idea is.

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/c1f398e84537a426bf51daed5f6738eca6248630/bigtable-protos/src/main/proto/google/bigtable/admin/table/v1/bigtable_table_data.proto#L29-L32

**Pull request merged**